### PR TITLE
DT-518 Simplify map server and add stop layer

### PR DESCRIPTION
**Pull request by hannesj**

### CODE REVIEW

File: menu.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ function printMenu {
     echo "9) Build Siri2GTFS-RT"
     echo "10) Build Digitransit-ui"
     echo "12) Build Postgre database"
-    echo "13) Build Vector Map Server"
+    echo "13) "
```
Comment (samuliheljo) [inline menu.sh:59]: Empty?

File: roles/map-server-docker-image/templates/Dockerfile.j2 [hunk 2]
```diff
@@ -10,18 +10,25 @@ WORKDIR {{map_server_root}}
...
+  tilelive-otp-stops \
+  tilelive-tmsource \
+  tilelive-tmstyle \
+  tilelive-vector \
+  tilelive-xray
 
 RUN git clone {{map_server_style_git}} {{map_server_style}}
+RUN git clone {{vector_map_server_style_git}} {{vector_map_server_style}}
+
+RUN sed -i -e 's#source: "http://tiletest.kapsi.fi:8080/index.json"#source: "tilejson+http://localhost:{{map_server_exposed_port}}/hsl-vector-map/index.json"#' {{map_server_style}}/project.yml
```
Comment (samuliheljo) [inline roles/map-server-docker-image/templates/Dockerfile.j2:25]: Could we move this index.json file to deploy project and send it to server with ansible?

File: roles/map-server-docker-image/templates/Dockerfile.j2 [hunk 3]
```diff
@@ -10,18 +10,25 @@ WORKDIR {{map_server_root}}
...
 
-RUN sed -i -e 's#source: "http://tiletest.kapsi.fi:8080/index.json"#source: "tilejson+http://vectormapserver:{{vector_map_server_exposed_port}}/hsl-vector-map/index.json"#' {{map_server_style}}/project.yml
+RUN sed -i -e 's/host: localhost/host: postgisosm/' {{vector_map_server_style}}/data.yml
+RUN sed -i -e 's/password: postgres/password: mysecretpassword/' {{vector_map_server_style}}/data.yml
```
Comment (samuliheljo) [inline roles/map-server-docker-image/templates/Dockerfile.j2:28]: password is very secret indeed :)
We should reset this at some point.

File: menu.sh (hunk 1 above)
Comment (hannesj) [inline menu.sh:59]: Yes, we no longer have a separate vector map server, do you have a better title in mind?

File: roles/map-server-docker-image/templates/Dockerfile.j2 (hunk 2 above)
Comment (hannesj) [inline roles/map-server-docker-image/templates/Dockerfile.j2:25]: No, it is just the address to the endpoint serving the vector map tiles.

**Pull request merged**